specification/src/main/asciidoc/chapters/Sparkplug_1_Introduction.adoc

```diff
@@ -51,10 +51,10 @@ solutions.
 ==== Define the MQTT Payload
 
 Just as the MQTT Specification does not dictate any particular Topic Namespace, it also does it
-dictate any particular payload data encoding. The intent of the Sparkplug Specification is to strive
-to define payload encoding architectures that remain true to the original, lightweight, bandwidth
-efficient, low latency features of MQTT while adding modern encoding schemes targeting the
-SCADA/IIoT solution space.
+dictate any particular payload data encoding. The intent of the Sparkplug Specification is to define
```

Comment:
Should it be: 'Just as the MQTT Specification does not dictate any particular Topic Namespace, it also does not dictate any particular payload data encoding.' ?